wfl/Models/FoodOrder.cs

```diff
@@ -16,9 +16,9 @@ public void OnDeserialize()
         {
             if (MenuSelections != null)
             {
-                foreach (MenuSelection selection in MenuSelections)
+                for(int i = 0; i < MenuSelections.Count; i++)
```

Comment:
This change is not necessary. I had changed this earlier when I was updating the index of each menu selection. I'll change it back.